Lombiq.HelpfulLibraries.Refit/Lombiq.HelpfulLibraries.Refit.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
https://lombiq.atlassian.net/wiki/spaces/DEV/pages/2609840133/Preparing+a+.NET+project+to+be+published+on+NuGet#:~:text=Add%20a%20NuGet%20version%20shield